### PR TITLE
Cloudformation: reorganize and clarify the form

### PR DESCRIPTION
**Pull request by benipeled**
This commit includes:
 - Re-order parameters by using AWS::CloudFormation::Interface
 - Add friendly name for parameters
 - A few description clarifications
 - Change the InstanceCount to be a string with list of 1-10

Fixes: #180 

@yaronkaikov please review.

**The form new design:**

image

### CODE REVIEW

File: aws/cloudformation/scylla.yaml.j2 [hunk 1]
```diff
@@ -10,37 +10,76 @@ Description: >-
...
+          - AvailabilityZone
+          - CIDR
+          - EnablePublicAccess
+          - SSHLocation
+          - KeyName
+    ParameterLabels:
+      ScyllaClusterName:
+        default: Scylla cluster name
+      ScyllaAmi:
+        default: Scylla AMI for the EC2 instances
```
Comment (yaronkaikov) [inline aws/cloudformation/scylla.yaml.j2:36]: AMI already indicates it's Ec2 instances. Maybe change it to Scylla AMI id?

File: aws/cloudformation/scylla.yaml.j2 [hunk 2]
```diff
@@ -10,37 +10,76 @@ Description: >-
...
+      SSHLocation:
+        default: Allowed subnet for public access (SSH)
+
 Parameters:
-  ClusterName:
+  ScyllaClusterName:
     Type: String
 
-  PublicCidrAccess:
+  SSHLocation:
```
Comment (yaronkaikov) [inline aws/cloudformation/scylla.yaml.j2:56]: Why Location?

**Review by yaronkaikov — CHANGES_REQUESTED** [review]

**Comment by yaronkaikov** [comment]
@benipeled Just a general comment, if you want me to review, you can just asign me as a reviewer (right side). if you can't let me know so i will fix it for you

File: aws/cloudformation/scylla.yaml.j2 (hunk 2 above)
Comment (benipeled) [reply aws/cloudformation/scylla.yaml.j2]: No good answer + it shouldn't be service specific (SSH in this case),
Changed to `PublicAccessCIDR`

**Review by yaronkaikov — APPROVED** [review]

**Pull request merged**